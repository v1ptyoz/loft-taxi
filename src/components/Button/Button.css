.btn {
  border: none;
  cursor: pointer;
  background: #FDBF5A;
  height: 60px;
  width: 100%;
  border-radius: 70px;
  font-size: 24px;
}
.btn:hover {
  background: #FFA842;
}
.btn[disabled] {
  background: #D8D7D5;
  color: #737373;
  cursor: default;
}
.btn[disabled]:hover {
  background: #D8D7D5;
}