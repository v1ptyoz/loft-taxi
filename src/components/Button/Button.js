import "./Button.css";

export function Button(props) {
  return (
      <button className="btn" disabled={props.disabled} onClick={props.handler}>{props.caption}</button>
  )
}