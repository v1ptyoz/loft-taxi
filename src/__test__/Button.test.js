import {Button} from "../components/Button/Button";
import renderer from "react-test-renderer";

describe("Button component", () => {
  it("Render normal via snapshot", () => {
    const button = renderer.create(<Button type="submit" disabled={true} caption="Button" />)
    expect(button).toMatchSnapshot();
  })
})