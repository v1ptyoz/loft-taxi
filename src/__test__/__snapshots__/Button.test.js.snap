// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Render normal via snapshot 1`] = `
<button
  className="btn"
  disabled={true}
  type="submit"
>
  Button
</button>
`;
